src/app/services/npm-registry/npm-registry.service.ts

```diff
@@ -1,99 +1,96 @@
-import { inject, Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
+import { inject, Injectable } from '@angular/core';
+import { format } from 'date-fns';
 import { catchError, forkJoin, map, Observable, of, tap } from 'rxjs';
 import { StorageService, StorageType } from '../storage';
-import { format } from 'date-fns';
 import { DownloadsPoint, DownloadsRange, RegistryData, Suggestion } from './npm-registry.model';
 
 @Injectable({
-  providedIn: 'root'
+	providedIn: 'root',
```

Comment:
github changes is showing this as 8 spaces for some reason. I assume this is github being weird or there actually is 8 spaces